linode/manager
# chore: Resolve `yaml` dependabot alert `CVE-2023-2251`

## Description 📝

- Resolve `yaml` dependabot alert CVE-2023-2251
- Removed some yarn resolutions that are no longer needed
- Updates `patch-package` and `lint-staged` because they depended on an old `yaml` but we still need a resolution thanks to the following packages
  - `react-select` depends on the old yaml version
  - `tsup` somehow still depends on  the old yaml version

## How to test 🧪
- Check basic repo functionality
- Verify that the yarn resolutions I removed do not result in a downgrade of those respective packages
- Consider testing `patch-package` and `lint-staged` because they were upgraded
  - I tested them as I made commits to this PR
